idea/tests/org/jetbrains/kotlin/idea/inspections/KotlinCleanupInspectionTest.kt

```diff
@@ -35,10 +35,14 @@ class KotlinCleanupInspectionTest() : KotlinLightCodeInsightFixtureTestCase() {
         val profile = InspectionProjectProfileManager.getInstance(project).currentProfile
         globalContext.codeCleanup(analysisScope, profile, "Cleanup", null, true)
 
-        myFixture.checkResultByFile(result)
+        myFixture.checkResultByFile("$dir/$result")
     }
 
-    fun testCleanup() {
-        doTest("cleanup.kt.after", "cleanup.kt", "JavaAnn.java", "deprecatedSymbols.kt")
+    fun testBasic() {
```

Comment:
You renamed `testCleanup` to `testBasic` but left `cleanup.kt` and `cleanup.kt.after` names. I guess they should be renamed to `basic.kt` and `basic.kt.after` as well